.travis.yml

```diff
@@ -1,16 +1,44 @@
 language: go
 go:
-  - 1.7.x
+  - "1.7.x"
 dist: trusty
 sudo: required
 group: edge
+
 before_install:
   - sudo apt-get -qq update
-  - sudo apt-get install -y autoconf pkg-config e2fslibs-dev libblkid-dev zlib1g-dev liblzo2-dev asciidoc
-install: ./scripts/travis.sh install
+  - sudo apt-get install -y python-apt autoconf pkg-config e2fslibs-dev libblkid-dev zlib1g-dev liblzo2-dev asciidoc
+install:
+  - sudo pip install ansible==2.3.1 docker-py
+  - ./scripts/travis.sh install
+before_script:
+  - tmp=`mktemp`
+  - echo '{"insecure-registries":["172.30.0.0/16"]}' > ${tmp}
+  - sudo mv ${tmp} /etc/docker/daemon.json
+  - sudo mount --make-shared /
+  - sudo service docker restart
 script:
   - ./scripts/travis.sh format
   - ./scripts/travis.sh vet
   - ./scripts/travis.sh lint
   - ./scripts/travis.sh build
   - ./scripts/travis.sh test
+  - ./scripts/travis.sh ci
+  - sudo docker pull docker.io/openshift/origin:v3.6.0-rc.0
+  - oc cluster up --image=docker.io/openshift/origin --version=v3.6.0-rc.0 --service-catalog=true
```

Comment:
If you don't specify the routing_suffix then I think you will see problems with the ServiceCatalog talking to the Ansible Service Broker.  The routing suffix defaults to 127.0.0.1.nip.io

The controller-manager pod will then attempt to access something like asb-1338-openshift-ansible-service-broker.apps.127.0.0.1.nip.io which goes back to itself 127.0.0.1 and is never able to connect to the broker.

We use the IP of the docker bridge on the host typically.

It's also possible we might be able to change how the broker registers to the service catalog, maybe we can use the kube dns between namespaces...not sure  but this implies it's possible to resolve services between namespaces:  https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/

Yet, also worth considering, even if the resolution works, there may be cases where namespaces are fully locked down and can not communicate at the SDN layer, isolated networks, so the route option perhaps is safest.